beacon-chain/state/fieldtrie/field_trie_helpers.go

```diff
@@ -60,6 +62,13 @@ func fieldConverters(field types.FieldIndex, indices []uint64, elements interfac
 				reflect.TypeOf([]*ethpb.PendingAttestation{}).Name(), reflect.TypeOf(elements).Name())
 		}
 		return handlePendingAttestation(val, indices, convertAll)
+	case types.Balances:
+		val, ok := elements.([]uint64)
+		if !ok {
+			return nil, errors.Errorf("Wanted type of %v but got %v",
+				reflect.TypeOf([]uint64{}).Name(), reflect.TypeOf(elements).Name())
+		}
+		return handleBalanceSlice(val, indices, convertAll)
```

Comment:
outside the scope of this PR, I'm noticing that the scope of these helpers is really inconsistent. `HandleEth1DataSlice` is  exported for no apparent reason, `HandleByteArrays` and `HandleValidatorSlice` are defined in a different package (`stateutil`) with no usages beyond the `fieldtrie` package.